lib/app.js

```diff
@@ -28,4 +28,15 @@ exports.getDocuments = getDocuments = function(callback) {
   });
 }
 
+exports.createChannelConfigObject = createChannelConfigObject = function(name, callback) {
+  var channelConfig = require('../resources/channelConfig.json');
```

Comment:
Just thinking about it again I think we will need to do something like this - https://github.com/jembi/perioperative-shared-health-record/blob/adf956161b321959103e5f8c6f0f8a0abfc3b4b8/test/patient.js#L15

Otherwise, changes that we make to `channelConfig` will persist across requires due to how the way require works. It only imports once and uses the same object when require is called again.